app/__init__.py

```diff
@@ -5,7 +5,7 @@
 from flask_wtf.csrf import CSRFProtect, generate_csrf
 from flask_login import LoginManager
 
-from .models import db, User
+from .models import db, User, Message, Post, Hashtag, PostLike, Comment, CommentLike, hashtagPostJoin, Follower
```

Comment:
maybe it is unnecessary, but we imported them all into our app/models/__init__.py. We thought it was necessary since we imported them here and db and User were both getting imported but there is no actual reason in particular haha.